src/core/org.freedesktop.systemd1.policy.in

```diff
@@ -70,4 +70,14 @@
                 </defaults>
         </action>
 
+        <action id="org.freedesktop.systemd1.acquire-namespace">
+                <description gettext-domain="systemd">Acquire pre-prepared namespace</description>
+                <message gettext-domain="systemd">Authentication is required to acquire pre-prepared namespace.</message>
```

Comment:
hmm, why do we actually need to restrict access to identity mappings?

Reply:
sounds like we do